core/processor/src/main/java/io/quarkus/annotation/processor/util/ExtensionUtil.java

```diff
@@ -20,10 +20,12 @@
 
 import io.quarkus.annotation.processor.documentation.config.model.Extension;
 import io.quarkus.annotation.processor.documentation.config.model.Extension.NameSource;
+import io.quarkus.annotation.processor.documentation.config.model.ExtensionModule;
+import io.quarkus.annotation.processor.documentation.config.model.ExtensionModule.ExtensionModuleType;
 
 public final class ExtensionUtil {
 
-    private static final String RUNTIME_MARKER_FILE = "META-INF/quarkus.properties";
+    private static final String RUNTIME_MARKER_FILE = "META-INF/quarkus-extension.properties";
```

Comment:
This was a dumb mistake on my side. It had no consequences until now but we need it to actually work from now on :).